src/ontology/mondo-edit.obo

```diff
@@ -40055,7 +40055,7 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "acquired polycythemia" EXACT [MONDO:patterns/acquired]
 synonym: "acquired polycythemia (disease)" EXACT [MONDO:patterns/acquired]
-synonym: "polycythemia, secondary" RELATED [DOID:2834]
+synonym: "polycythemia, secondary" RELATED EXCLUDE [DOID:2834]
```

Comment:
I don't follow the trail from the original ticket to merge two terms to adding `EXCLUDE` as a [synonym type](https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#synonym-types)

Reply:
Ha - I linked the wrong ticket, so that is why it did not make sense! I fixed it now. Thanks!